node/src/chain_spec.rs

```diff
@@ -264,6 +268,50 @@ pub fn statemine_local_config(id: ParaId) -> StatemineChainSpec {
 	)
 }
 
+pub fn statemine_mainnet_config(id: ParaId) -> StatemineChainSpec {
+	//TODO check if correct
+	let data = r#"
+		{
+			"ss58Format": 1,
```

Comment:
@joepetrowski

Reply:
...also we want 12 decimals to match KSM and ksm symbol?

Reply:
SS58: 2
Decimals: 12